source/main.cpp

```diff
@@ -136,6 +136,8 @@ int main(int argc, char *argv[])
 	// Whether we are running an integration test.
 	const bool isTesting = !testToRunName.empty();
 	try {
+		PlayerInfo player;
```

Comment:
This variable looks unused when looking at the diff. Is this variable actually used?

Reply:
Looks like it was made unnecessary by a merge from master.